components/alternatehreflangelements/default.htm

```diff
@@ -0,0 +1,3 @@
+{% for locale, alternateUrl in __SELF__.locales %}
+    <link rel="alternate" hreflang="{{ locale }}" href="{{ url(alternateUrl) }}" />
```

Comment:
What does the `url()` function do? Does it actually work? I've never used it in October. Why not build the full URL in the PHP code that's creating the `alternateUrl` value and just use `{{ alternateUrl }}`?

Reply:
It is a helper for  [`URLGenerator->to($path)`](https://github.com/laravel/framework/blob/5.5/src/Illuminate/Routing/UrlGenerator.php#L170)

Reply:
So long as it works I'm fine with it then :) Thanks for your work on it! Is this ready to merge now?

Reply:
It spits out `https://build426.dev/en/tesssst` instead of `/en/tesssst` :)
So lets merge! If it creates a problem somehow we can adjust it easily.